Networking/Networking/Model/Copiable/Models+Copiable.generated.swift

```diff
@@ -2,6 +2,7 @@
 // DO NOT EDIT
 import Codegen
 import Foundation
+import WooFoundation
```

Comment:
I'm not sure why this import was added, but I'm leaving it since it was a result of running `rake generate`.